drivers/mpsl/flash_sync/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if SOC_FLASH_NRF
+
+choice SOC_FLASH_NRF_RADIO_SYNC_CHOICE
+	prompt "Nordic nRFx flash driver synchronization"
+	default SOC_FLASH_NRF_RADIO_SYNC_MPSL if MPSL
+
+config SOC_FLASH_NRF_RADIO_SYNC_MPSL
+	bool "Nordic nRFx flash driver synchronized using MPSL"
+	depends on MPSL
+	help
+	  Enable synchronization between flash memory driver and MPSL.
+
+endchoice
+
+module = SOC_FLASH_NRF_RADIO_SYNC_MPSL
+module-str = flash_sync_mpsl
+source "subsys/logging/Kconfig.template.log_config"
+
+endif
+
+config SOC_FLASH_NRF_RADIO_SYNC_MPSL_TIMESLOT_SESSION_COUNT
```

Comment:
Do we need a separate config for this? The flash driver will always use one sesssion

Reply:
Yes, but the code in `mpsl_init.c` is easier if this specifies 0 or 1.

Reply:
I see your point. Still I would prefer to not have this config. It indicates to the reader that the value can be greater than 1

Reply:
The reader would not be able to modify this anyway.

Reply:
I mean modify using menuconfig or prj.conf.